MM/6.0.0/config/media_manager_5/labels/Youtube.dcl

```diff
@@ -318,38 +318,6 @@ resource configservice_label youtube_client_info_client_secret_validation_failed
   ]
 }
 
-resource configservice_label youtube_client_info_search_name {
-  key = 'YOUTUBE_CLIENT_INFO_SEARCH_NAME'
-  group = 'Youtube'
-  product_id = resource.configservice_product.media_manager_5.id
-  default_label_values = [
-    {
-      default_translation = 'Search name'
-      language_id = data.language.english.id
-    },
-    {
-      default_translation = 'Navn på søgning'
-      language_id = data.language.danish.id
-    }
-  ]
-}
-
-resource configservice_label youtube_client_info_search_name_validation_failed_required {
-  key = 'YOUTUBE_CLIENT_INFO_SEARCH_NAME_VALIDATION_FAILED_REQUIRED'
-  group = 'Youtube'
-  product_id = resource.configservice_product.media_manager_5.id
-  default_label_values = [
-    {
-      default_translation = 'Search name is required'
-      language_id = data.language.english.id
-    },
-    {
-      default_translation = 'Navn på søgning er påkrævet'
-      language_id = data.language.danish.id
-    }
-  ]
-}
```

Comment:
We should consider adding a migration in Configuration Management, similar to the existing `RemoveLabelsMigration_5_10_0`, to ensure that we don't run into issues if customers have translated these labels.